build.sbt

```diff
@@ -28,7 +28,8 @@ lazy val commonSettings = Seq(
   testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-h", "target/test-reports"),
   coverageEnabled in Test := true,
   coverageMinimum := 60,
-  coverageFailOnMinimum := true
+  coverageFailOnMinimum := true,
+  coverageExcludedPackages := "CromwellPipelineApp\\..*"
```

Comment:
Maybe we should put more entities here :)

Reply:
Definitely. We should exclude from coverage everything in `portal` module.